nightwatch/commands/initializeNewUser.js

```diff
@@ -4,12 +4,17 @@ exports.command = function (user) {
     var self = this;
     
     self.perform(function () {
-        var fullName = 'nightwatch+' + user.name + '+' +
-            self.globals.timeTestStarted;
+        // todo: code could break if tests are run in parallel and the timeTestStarted
```

Comment:
For future reference, a "to do" tag is usually capitalized to standout. So, `// TODO: <blah>`